server.go

```diff
@@ -1614,7 +1620,12 @@ func (s *server) establishPersistentConnections() error {
 		if ok {
 			for _, lnAddress := range linkNodeAddrs.addresses {
 				switch lnAddress.(type) {
-				case *net.TCPAddr, *tor.OnionAddr:
+				case *net.TCPAddr:
+					addrSet[lnAddress.String()] = lnAddress
+
+				// We'll only attempt to connect to Tor
+				// addresses if Tor outbound support is enabled.
+				case *tor.OnionAddr:
```

Comment:
should this also check `cfg.Tor.Active`?

Reply:
Fixed.